### ISSUE TITLE
Rescan for devices?

### ISSUE THREAD

**Issue by sdaau** [linuxmint/warpinator#172]
I just built warpinator from git on Ubuntu 22.04. I have also installed it on my android device from https://f-droid.org/en/packages/slowscript.warpinator/ .

I have set the same  group code on both. Now I want these two devices to find each other - but I cannot tell how to do that on Ubuntu build, short of restarting the program (I hope, that is not the idea?)

In the android build, in the "three vertical dots" menu top right, there is an option "Reannounce", as well as "Rescan for devices"; but I cannot find anything like that in the Ubuntu build?

So, basically, since I started from command line, I could see on the Ubuntu warpinator log in terminal:

```
2023-04-13 21:58:12,486::warpinator: >>> Discovery: new remote: MY-PHONE (192.168.X.XX:42000)
```

... but there was absolutely no device shown there, for some minutes after the Discovery message was printed - so it looked something like this:

![image](https://user-images.githubusercontent.com/26737749/231874315-63537c79-97a7-47e4-9a11-08f88c82bf2c.png)

So, how do I do a "rescan for devices" on the Ubuntu GUI?

**Comment by baozidai** [comment]
On my Linux Mint 21.1, it will auto refresh and appear when your phone get online, and that means don't need a manual "rescan for devices".

**Comment by HeidrHH** [comment]
I have the same problem with my Android smartphone: 

S21, SM-G99B/DS, One UI-Version 5.1, Android Version 13

The problem shows on both computers, different hardware but same Linux:

Operating System: TUXEDO OS 2 (based on Ubuntu 22.04 2)
KDE Plasma Version: 5.27.7
KDE Frameworks Version: 5.108.0
Qt Version: 5.15.10
Kernel Version: 6.2.0-10018-tuxedo (64-bit)
Graphics Platform: X11

If I tap on my smartphone on the symbol with the 2 arrows I get the following message:

"Connected, service not available."

Both computer see the other computer but the Smartphone shows as disconnected on both. There is no automatic reconnection to the Smartphone.

I don't have a firewall on my computer, the firewall is in my router Fritz!box.

I can only do port sharing to the Internet. Behind the firewall, the devices should be able to communicate normally, if I understand correctly.

**Comment by SuperUserNameMan** [comment]
I agree that a "refresh" button would be really appreciated.

**Comment by mtwebster** [comment]
Implemented in https://github.com/linuxmint/warpinator/commit/a81c39d7786610fcf9cd6b1e536f66137970df2f
